react/styling-and-css.md

```diff
@@ -128,6 +128,29 @@ Component styles are declared in a separate `.styles.js` file using the [`css` h
 
 [SMACSS categories](https://smacss.com/book/categorizing) provide a practical separation of concerns. Module and state categories are covered by the `component.style.js` pattern, however styles that reach accross components (whether because they are global or just affect many components) need a proper place. Place them in `src/common_styles` and use SMACSS catogires for grouping. Applies both to CSS and CSS-in-JS.
 
+#### Naming
+
+Elements within the style files use the `Style` suffix.
+
+```js
+// => Component.style.js
+import { css } from "styled-components";
+
+export const buttonStyle = css`
+  // ...
+`;
+```
+
+Styled components apply *UpperCamelCase* without the suffix.
```

Comment:
I think `UpperCamelCase` already comes from other naming guides. It'd be enough to say that 

```suggestion
Styled components don't use the suffix.
```

Reply:
Great idea, thanks!